docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
+
+[Guzzle](http://docs.guzzlephp.org/en/latest/) is a library for accessing URL's through PHP's [cURL library](http://php.net/manual/book.curl.php). Guzzle is automatically installed with Codeception. You can easily access it from any test file:
+
+```php
+// ExternalCest.php
```

Comment:
I left it out on purpose. I copied the style from Symfony's code blocks, see e.g. http://symfony.com/doc/current/controller.html

Reasons:

* That a PHP framework is written in PHP should be obvious :-)

* More important is the question "In which file is this supposed to go?" That's why the file name should be included in every code block.

But if you *really* want the `<?php`, I'd write it like that:
```php
<?php
// ExternalCest.php
```